.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,31 @@
+name: Bug Report
+description: Create a bug report to help us fix issues with the Quilt website.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to report this bug! If you need real-time help, join us on [Discord](https://discord.quiltmc.org).
+
+  - type: textarea
```

Comment:
Can you add separate fields for "Steps to reproduce", "Expected results" and "Actual Results"?